packages/resource-deployment/scripts/install-parallel.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# shellcheck disable=SC1090
+set -eo pipefail
+
+export appInsightsKey
+export apiManagementName
+export batchAccountName
+export cosmosAccountName
+export datalakeStorageAccountName
+export dropFolder="${0%/*}/../../../"
+export environment
+export functionAppName
+export keyVault
+export keyVaultUrl
+export location
+export resourceGroupName
+export subscription
+export storageAccountName
+export clientId
+export templatesFolder="${0%/*}/../templates/"
+export apiTemplates="$templatesFolder"rest-api-templates
+
+exitWithUsageInfo() {
+    echo "
+Usage: $0 -e <environment> -l <Azure region> -o <organisation name> -p <publisher email> -r <resource group> -s <subscription name or id> -c <client id>
+where:
+Resource group - The name of the resource group that everything will be deployed in.
+Subscription - The subscription for the resource group.
+Environment - The environment in which the set up is running.
+Publisher email - The email for notifications.
+Resource group - The resource group that this API instance needs to be added to.
+Client ID - the app registration client id used for function app authentication/ authorization.
+Azure region - Azure region where the instances will be deployed. Available Azure regions:
+    centralus
+    eastasia
+    southeastasia
+    eastus
+    eastus2
+    westus
+    westus2
+    northcentralus
+    southcentralus
+    westcentralus
+    northeurope
+    westeurope
+    japaneast
+    japanwest
+    brazilsouth
+    australiasoutheast
+    australiaeast
+    westindia
+    southindia
+    centralindia
+    canadacentral
+    canadaeast
+    uksouth
+    ukwest
+    koreacentral
+    koreasouth
+    francecentral
+    southafricanorth
+    uaenorth
+"
+    exit 1
+}
+
+
+function waitForProcesses(){
+    processesToWaitFor=$1
```

Comment:
Not sure we need script exit code, we can drop return/echo and just wait for processes. Will wait fail if process already was terminated before we call wait for that process id? We may change the wait call then similar to: 

wait $pid 2>/dev/null || true

Reply:
Discussed offline. #NoAction